src/applications/mhv-secure-messaging/main/tests/e2e/keyboard-nav-test/secure-messaging-keyboard-nav-to-message-details.cypress.spec.js

```diff
@@ -27,6 +27,45 @@ describe('Navigate to Message Details ', () => {
     cy.realPress('Tab');
     cy.get(Locators.BUTTONS.BUTTON_TRASH).should('be.visible');
 
+    cy.injectAxe();
+    cy.axeCheck(AXE_CONTEXT, {
+      rules: {
+        'aria-required-children': {
+          enabled: false,
+        },
+        'color-contrast': {
+          enabled: false,
+        },
+      },
+    });
+  });
+  it('Keyboard Nav Access to Expended Messages', () => {
+    const landingPage = new PatientInboxPage();
+    const messageDetailsPage = new PatientMessageDetailsPage();
+    const site = new SecureMessagingSite();
+    site.login();
+    mockMessagewithAttachment.data.id = '7192838';
+    mockMessagewithAttachment.data.attributes.attachment = true;
+    mockMessagewithAttachment.data.attributes.body = 'attachment';
+    landingPage.loadInboxMessages(mockMessages, mockMessagewithAttachment);
+    messageDetailsPage.loadMessageDetails(mockMessagewithAttachment);
+    cy.contains('Print').should('be.visible');
+    cy.tabToElement('button')
+      .eq(0)
+      .should('contain', 'Print');
+    cy.realPress('Tab');
+    cy.get(Locators.BUTTONS.BUTTON_MOVE).should('have.focus');
+    cy.realPress('Tab');
+    cy.get(Locators.BUTTONS.BUTTON_TRASH)
+      .should('be.visible')
+      .then(() => {
+        cy.get(Locators.BUTTONS.BUTTON_TRASH).should('have.focus');
+      });
+    for (let i = 0; i < 9; i++) {
+      cy.realPress('Tab');
+    }
```

Comment:
I think I understand the goal this loop is trying to reach, which is tab to "Expand All +" button. I am concerned that an arbitrary number of iterations here may not be reliable. is there a way to do a `while` loop with a condition when a specific button reaches focus?